content/pages/sphinx.html

```diff
@@ -0,0 +1,243 @@
+{% import "partials/technical.html" as technical %}
+
+<section>
+  {% call basic.header_image(
+    header="Quickly deploy your Sphinx docs across multiple versions",
+    image="https://cdn.dribbble.com/users/876899/screenshots/2141696/readthedocs-thumb.png",
+    image_alt="Marketing introduction header",
+    vertical_align="middle")
+  %}
+    <p>
+        Beautiful docs for your team are just a <code>git push</code> away.
+    </p>
+
+    <p class="ui basic padded vertical segment">
+      <a href="#" class="ui large stackable primary button">
+        Register an account
+      </a>
+      <a href="https://docs.readthedocs.io/en/stable/tutorial/index.html" class="ui large stackable button">
+        Start our tutorial
+      </a>
+    </p>
+  {% endcall %}
+</section>
+
+<section id="features">
+  {% call marketing.highlight_list() %}
+    {% call marketing.highlight_item(
+        icon="olive fa-cauldron",
+        icon_style="--fa-secondary-color: violet; --fa-secondary-opacity: 1;",
+        header="Preview each pull request",
+        color="primary") %}
+    {% markdown %}
+    Every time you create a new Pull Request on GitHub or GitLab,
+    we rebuild your docs and give you a live preview of the changes.
+    **Documentation review has never been this easy**.
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="yellow fa-paintbrush",
+        icon_style="--fa-secondary-color: white; --fa-secondary-opacity: 1;",
+        header="Customize the theme & domain",
+        color="secondary") %}
+    {% markdown %}
+    You can host the docs on your own domain (``docs.company.com``),
+    and fully customize the HTML theme.
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="fa-image-landscape",
+        header="Multiple docs versions",
+        color="violet") %}
+    {% markdown %}
+    Host the docs for all the supported versions of your software,
+    so that users can access the exact one they need.
+    Works great for different development workflows,
+    as well as public documentation.
+    {% endmarkdown %}
+        {% endcall %}
+    {% endcall %}
+</section>
+
+<div class="ui very padded container">
+
+{{ basic.header_icon(
+  header="Some of our users",
+  subheader="Thousands of businesses and open source projects use Read the Docs daily",
+  icon="purple fa-message-lines",
+  icon_style="--fa-secondary-color: blueviolet;"
+) }}
+
+{#
+  Cards for an articles or case studies view
+
+  See https://fomantic-ui.com/views/card.html
+#}
+<div class="ui three link stackable cards">
+    <a href="https://flask.palletsprojects.com/en/2.2.x/" class="card">
+    <div class="ui small centered image">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/3/3c/Flask_logo.svg" alt="A random image">
+      </div>
+      <div class="content">
+        <div class="header">Flask</div>
+        <div class="meta">
+          Case study
```

Comment:
These ones could be a category, like 

- Web framework
- Scientific layer for Sphinx
- Distributed file system

or similar...